### ISSUE TITLE
set-mark-command is not working after using expand-region

### ISSUE THREAD

**Issue by ludwigpacifici** [magnars/expand-region.el#222]
Hello,

with `emacs -Q`, I open a C++ file and I do `C-SPC C-n`:

![tmp14](https://cloud.githubusercontent.com/assets/923945/23120047/aecb14a8-f752-11e6-8980-6d6b2254692a.png)

It's all good. Then, I load expand region, with this snippet:

```
(add-to-list 'load-path "~/.emacs.d/elpa/expand-region-20170213.616")
(require 'expand-region)
(global-set-key (kbd "C-=") 'er/expand-region)
```

Then I do `C-=`:

![tmp15](https://cloud.githubusercontent.com/assets/923945/23120229/6079ed3c-f753-11e6-9150-d6874af9875e.png)

That's correct. Then `C-g C-SPC C-n`:

![tmp16](https://cloud.githubusercontent.com/assets/923945/23120242/6cd5db7c-f753-11e6-9323-1a9e752ee408.png)

I do see anymore the selection like in the first image. Is it expected?

Basically, my workflow to select text relies on `C-SPC` and `expand-region`. But it looks like they do not work well together.

`M-x version`: `GNU Emacs 26.0.50.1 (x86_64-unknown-linux-gnu, GTK+ Version 2.24.28) of 2017-02-13`

**Comment by ludwigpacifici** [comment]
I just saw, it might be a duplicate of #220
